src/TwilioVideo.ios.js

```diff
@@ -148,19 +148,6 @@ export default class extends Component {
 
     this._subscriptions = []
     this._eventEmitter = new NativeEventEmitter(TWVideoModule)
-
-    this.setLocalVideoEnabled = this.setLocalVideoEnabled.bind(this)
-    this.setLocalAudioEnabled = this.setLocalAudioEnabled.bind(this)
-    this.flipCamera = this.flipCamera.bind(this)
-    this.connect = this.connect.bind(this)
-    this.disconnect = this.disconnect.bind(this)
-    this.sendString = this.sendString.bind(this)
-    this.setRemoteAudioPlayback = this.setRemoteAudioPlayback.bind(this)
-    this.unpublishLocalAudio = this.unpublishLocalAudio.bind(this)
-    this.unpublishLocalVideo = this.unpublishLocalVideo.bind(this)
-
-    this.publishLocalAudio = this.publishLocalAudio.bind(this)
-    this.publishLocalVideo = this.publishLocalVideo.bind(this)
```

Comment:
why do we need to remove this? then we should change those as arrow function, right?

Reply:
`.bind` or arrow functions are only needed when the method is using `this` inside of it, the methods whose binds were removed here were not accessing `this` anywhere but referencing a static var from the top, they should unaffected. If you build and run this on iOS, it'll work flawlessly

Reply:
Yeah. Got it. Thanks